modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java

```diff
@@ -5569,8 +5575,9 @@ public UserModel newSampleUserModel() {
 		_sampleUserId = _counter.get();
 
 		return newUserModel(
-			_sampleUserId, _SAMPLE_USER_NAME, _SAMPLE_USER_NAME,
-			_SAMPLE_USER_NAME, UserConstants.TYPE_REGULAR);
+			_counter.get(), "default-service-account",
```

Comment:
@slnn Why do you change the `newSampleUserModel()` to generate the `default-service-account`?

Reply:
Sorry, my bad, will resend